documentation/source/newsfragments/1693.removal.rst

```diff
@@ -0,0 +1 @@
+Makefile and documentation for the NVC simulator which hasn't been working since some time have been removed.
```

Comment:
```suggestion
Support for the NVC simulator has been removed.
```

Reply:
I initially had it like that, but this sounds as if there ever was "support" to speak of in the first place, which apparently there wasn't. "Non-support has been  removed" would be correct :)